mongo/integration/mtest/mongotest.go

```diff
@@ -685,8 +685,8 @@ func verifyServerParametersConstraints(serverParameters map[string]bson.RawValue
 	return nil
 }
 
-// verifyRunOnBlockConstraint returns an error if the current environment does not match the provided RunOnBlock.
-func verifyRunOnBlockConstraint(rob RunOnBlock) error {
+// VerifyRunOnBlockConstraint returns an error if the current environment does not match the provided RunOnBlock.
+func VerifyRunOnBlockConstraint(rob RunOnBlock) error {
```

Comment:
Is this publicizing just an overlooked function from [exporting](https://github.com/mongodb/mongo-go-driver/commit/b7f9264bd8759ee6f922dca899daa4d4a0009ec2) the unified test runner?